Sources/NIOHTTPCompression/HTTPResponseCompressor.swift

```diff
@@ -84,11 +127,21 @@ public final class HTTPResponseCompressor: ChannelDuplexHandler, RemovableChanne
     private var pendingWritePromise: EventLoopPromise<Void>!
 
     private let initialByteBufferCapacity: Int
+    private let responseCompressionPredicate: ResponseCompressionPredicate?
 
-    /// Initialise a ``HTTPResponseCompressor``
+    /// Initialize a ``HTTPResponseCompressor``.
+    /// - Parameter initialByteBufferCapacity: Initial size of buffer to allocate when hander is first added.
+    public convenience init(initialByteBufferCapacity: Int) {
```

Comment:
Hmm, when I suggested taking off the default argument value I actually meant to take it from the other initializer's `responseCompressionPredicate`. But I now wonder: why can't we have default values here? There should be no ambiguity, the compiler should be fine.

Reply:
Initially I ran into trouble, but I forgot what the issue was — let me give it another try

Reply:
You're right, we can have default arguments for both of them after all, let me add it back